config/version.txt

```diff
@@ -1 +1 @@
-0.2.0
+0.2.0+qa
```

Comment:
Are these changes until our CI accepts code

Reply:
This time I looked up what the version requirements are. Since PEP440 takes a stricter view of what is allowed in the public version I'm adding `qa` as a local tag, then when red signs off, we'll promote to `0.2.0rc1` which product will evaluate.

Reply:
Good to know; I was not familiar with PEP440.